modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/TableCell.js

```diff
@@ -77,10 +77,19 @@ function TableCell({
 }) {
 	let dataRenderer = DefaultRenderer;
 
-	const {inlineEditingSettings} = useContext(DataSetContext);
+	const {customDataRenderers, inlineEditingSettings} = useContext(
+		DataSetContext
+	);
+
+	const contentRenderer = view.contentRenderer;
 
-	if (view.contentRenderer) {
-		dataRenderer = getDataRendererById(view.contentRenderer);
+	if (contentRenderer) {
+		if (customDataRenderers && customDataRenderers[contentRenderer]) {
+			dataRenderer = customDataRenderers[contentRenderer];
+		}
+		else {
+			dataRenderer = getDataRendererById(view.contentRenderer);
```

Comment:
this is a very very minor thing but I would rename`getDataRendererById` to `getDefaultDataRendererById`. It will be more clear that we have default(predefined ones) and custom renderers

Reply:
Hmm I think `getDefaultDataRendererById` might be confusing, because we have the `DefaultRenderer`. It is fetched if you say `getDataRendererById('default')`. A dev might expect that `getDefaultDataRendererById` only returns the default renderer.